src/Helpers/UtilityFunctions.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * Scaffold Extension Utility Functions
+ */
+
+namespace Pantheon\TerminusScaffoldExtension\Helpers;
+
+class UtilityFunctions
+{
+    public const UPDATE_VARS_COLOR = "\e[38;5;3m";
+    public const DEFAULT_COLOR = "\e[0m";
+
+    /**
+     * Given a $site_env which might be in site_id.dev or just site_id format, return an array with the site_id and env.
+     *
+     * The default environment is dev.
+     *
+     * @param string $site_env
+     *
+     * @return array
+     */
+    public static function decypherSiteInfo(string $site_env) : array
```

Comment:
Is this not already something in core terminus lib? or is defaulting to dev the fun edge case?

Reply:
`getSite` from `SiteAwareInterface` expects us to know the `$site_id` already, and `getEnvironments` expects us to know the `$site_env`. When I saw these being invoked elsewhere, it looked to me like this stuff still had to be identified somehow, so I think this helper function is still useful. See https://github.com/pantheon-systems/terminus-secrets-manager-plugin/blob/main/src/Commands/SiteSetCommand.php#L59-L61